snuba/consumers/consumer.py

```diff
@@ -590,13 +591,35 @@ def process_message(
         return result
 
 
+def process_message_multistorage_work(
```

Comment:
Since this is not supposed to be used externally. Let's make it "private" ?
`_process_message_multistorage_work`